core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -238,13 +238,15 @@ export class TranslationModalComponent {
   }
 
   computeTranslationEditorOverflowState(): void {
-    const windowHeight = this.wds.getHeight();
-    const heightLimit = windowHeight * (this.cutoff_height) / 100;
+    setTimeout(() => {
```

Comment:
https://stackoverflow.com/a/48216423 suggests this is related to the Angular lifecycle hook. Maybe it's better to use something like this: https://stackoverflow.com/a/45350792 to recalculate the window heights?

Reply:
Hi Sagang, thanks for your suggestion.

I tried it in localhost like this:
<img width="534" alt="Screenshot 2023-02-27 at 11 19 16 PM" src="https://user-images.githubusercontent.com/2841740/221781830-83c354ff-f3e4-46b3-bbde-6fda9d30191c.png">
But still getting the reported error:
<img width="426" alt="Screenshot 2023-02-27 at 11 18 02 PM" src="https://user-images.githubusercontent.com/2841740/221781931-f2b3c876-f327-4f2f-b6ce-3d7cd8690fc6.png">
(I am not sending that commit as it is failing a frontend test too)

Would the setTimeout solution be inelegant? Is there a more elegant solution?

Reply:
It feels a bit clunky to me to have to wait a 1ms to update the values. Do you know what is causing the issue in the first place? I think it would be good to dig into exactly why Angular lifecycle hooks is throwing this and fix the root cause.

Reply:
Got it, I now applied a solution with ngAfterContentChecked which does the trick.

I discarded this solution when I initially started working on this issue because I saw still a similar error, however that second error was related but not exactly the same. It is also fixed in the newest commit.